src/rust/engine/options/src/args_tests.rs

```diff
@@ -59,7 +64,11 @@ fn test_bool() {
         "--baz=true",
         "--baz=FALSE",
         "--no-spam-eggs=False",
-        "--no-b=True",
```

Comment:
This is the one case we no longer support - we never supported it in Python, and it doesn't really make sense. Short flags are intended to be a single letter prefixed with a single `-`, and this is not that.

Also, in practice we only have one short flag left (`-l`) and we should probably deprecate that...

Reply:
> Also, in practice we only have one short flag left (-l) and we should probably deprecate that...

👍